apps/projects/app/components/Panel/usePanelManagement.js

```diff
@@ -42,11 +42,16 @@ const usePanelManagement = () => {
       setPanelProps({
         issue,
         requestIndex,
+        title: 'View application'
       })
     },
     reviewWork: (issue, index = 0) => {
       setActivePanel(PANELS.ReviewWork)
-      setPanelProps({ issue, index })
+      setPanelProps({
+        issue,
+        index,
+        title: 'View work'
+      })
```

Comment:
Probably no need to break this onto multiple lines; even on one line it looks like it would be less than 80 characters long. Up to you, though.

Reply:
Left it broken, because all other `setPanelProps` are this way.